nixpkgs_review/report.py

```diff
@@ -147,17 +193,22 @@ def markdown(self, pr: Optional[int]) -> str:
         if pr is not None:
             cmd += f" pr {pr}"
 
-        msg = f"Result of `{cmd}` run on {self.system} [1](https://github.com/Mic92/nixpkgs-review)\n"
+        shortcommit = f" at {self.pr_rev[:8]}" if self.pr_rev else ""
+        link = "[1](https://github.com/Mic92/nixpkgs-review)"
+        msg = f"Result of `{cmd}`{shortcommit} run on {self.system} {link}\n"
 
-        msg += html_pkgs_section(self.broken, "marked as broken and skipped")
+        msg += html_pkgs_section(self.broken, "marked as broken and skipped", show=10)
```

Comment:
I think we shouldn't limit the contents of lists in details by default.

Reply:
When I run nixpkgs-review against staging in `r-rmcgibbo`, there are sometimes up to 10,000 attrs "skipped" by the pre-build-hook filter, so I think some amount of limiting is necessary. Perhaps it can apply only to the "skipped" category rather than the "broken" category?

Reply:
Currently we don't list skipped packages at all so we did not have this problem. I think applying it to skipped packages is fine I think.